src/utils.js

```diff
@@ -324,7 +324,7 @@ function escapeLocalIdent(localident) {
   );
 }
 
-function defaultGetLocalIdent(
+export function defaultGetLocalIdent(
```

Comment:
We should not export it here, we should reexport it from the entry point, i.e. `index.js`

Reply:
It is done here https://github.com/webpack-contrib/css-loader/pull/1423/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R30

Reply:
We should not export it from `utils.js`, we can rename file or move it to another places, just reexport it from `index.js`

Reply:
This is exactly what i did. But re-export in index it should be exported from utils too

Reply:
ready to review. Sorry for the mess